Add docs about Personal Access Tokens (#2641)
flowforge/flowforge: docs/api/README.md

### Accessing the API

To make use of the API you will need a valid Access Token. Tokens can be generated 
for a user under the Security section of the User Settings page.

![Tokens Settings Page](images/tokens.png)

Tokens can be set to have a limited life or unlimited and can be revoked
by deleting the token from the list. Tokens with an expiry date will be deleted once they reach that date.

Be aware that the token value will only be displayed once, at creation time,
there is no way to recover the token after this point.

Currently, all routes require a valid token to be included in the request.

The tokens are passed using the `Authorization` header as a `Bearer` token.

> e.g. the following will get a list of all teams the user who owns the 
> token is a member of

Reviewer: ```suggestion
For example, the following will get a list of the token owner's teams:
```
